File: desktop/src/com/gamejam/helth/desktop/DesktopLauncher.java
package com.gamejam.helth.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.gamejam.helth.Helth;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.title = "Helth";
		config.width = 800;
		config.height = 480;
		new LwjglApplication(new Helth(), config);
	}
}

File: desktop/src/com/gamejam/helth/desktop/DesktopLauncher.kt
package com.gamejam.helth.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.gamejam.helth.Helth

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.title = "Helth"
        config.width = 800
        config.height = 480
        LwjglApplication(Helth(), config)
    }
}